src/test/java/org/gridsuite/modification/server/Impacts/TestImpactUtils.java

```diff
@@ -217,6 +210,11 @@ public static List<AbstractBaseImpact> createMultipleDeletionImpacts(List<Pair<I
         return new ArrayList<>(deletedIdentifiables.stream().map(identifiable -> createDeletionImpactType(identifiable.getLeft(), identifiable.getRight(), impactedSubstationIds)).toList());
     }
 
+    public static List<AbstractBaseImpact> createSubstationImpacts(Set<String> substationIds) {
+        return substationIds.stream().map(id -> createElementImpact(SimpleImpactType.MODIFICATION, IdentifiableType.SUBSTATION, id, Set.of(id)))
+                                    .collect(Collectors.toList());
```

Comment:
```suggestion
                                    .toList();
```

Reply:
It doesn't work because I must return a `List<AbstractBaseImpact>` and build a `List<SimpleElementImpact>`
What do you think about using `List.copyOf()` in this situation ? Which does not necessarily make a copy.